run_all_experiments.py

```diff
@@ -351,6 +358,47 @@ def add_to_json_report(outdir: str, key: str, value: Any) -> None:
     f.write(json.dumps(json_report))
 
 
+def _process_total_coverage_gain(results: list[Result]) -> dict[str, float]:
+  """Processes and calculates the total coverage gain for each project."""
+  textcov_dict: dict[str, list[textcov.Textcov]] = {}
+  for result in results:
+    if not isinstance(result.result, run_one_experiment.AggregatedResult):
+      continue
+    cov = result.result.gained_textcov
+    if not cov:
+      continue
+    if result.benchmark.project in textcov_dict:
+      textcov_dict[result.benchmark.project].append(cov)
+    else:
+      textcov_dict[result.benchmark.project] = [cov]
+
+  coverage_gain: dict[str, float] = {}
+  for project, cov_list in textcov_dict.items():
```

Comment:
Are we subtracting existing oss-fuzz coverage somewhere? We need to do this to get meaningful results. 

See https://github.com/google/oss-fuzz-gen/blob/main/report/aggregate_coverage_diff.py.

Reply:
The textcov used in here has already been subtracted. They are the textcov reports for the line diff coverage gain for each of the targets.